Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.cpp

```diff
@@ -652,7 +644,7 @@ namespace AZ::Reflection
                 if (HandleNodeInParentGroup(nodeData, parentData))
                 {
                     m_nodeWasSkipped = true;
-                    return true;
+                    return false;
```

Comment:
I know these magic bools were not done by you, but would you mind putting a comment here as to why we're returning false instead of true?

Reply:
Done!